src/packet_analysis/protocol/arp/ARP.cc

```diff
@@ -0,0 +1,19 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "ARP.h"
+
+using namespace zeek::packet_analysis::ARP;
+
+ARPAnalyzer::ARPAnalyzer()
+	: zeek::packet_analysis::Analyzer("ARP")
+	{
+	}
+
+zeek::packet_analysis::AnalyzerResult ARPAnalyzer::Analyze(Packet* packet, const uint8_t*& data)
+	{
+	// TODO: Make ARP analyzer a native packet analyzer
+	packet->l3_proto = L3_ARP;
+
+	// Leave packet analyzer land
```

Comment:
This would be good make more explicit I think, as it's hard to see the difference between terminating due to an error, and terminating due moving on to the next layer. I'll add more on this my note on the signature for `Analyze()`.

Reply:
See comments above.